js/ccf-app/typedoc.json

```diff
@@ -11,7 +11,10 @@
     "src/polyfill.ts"
   ],
   "out": "html",
-  "theme": "./doc/theme",
+  "theme": "default",
```

Comment:
The old custom theme no longer works, and it's not entirely clear how the new mechanism is meant work either: https://github.com/TypeStrong/typedoc/blob/master/internal-docs/custom-themes.md

The version selector wasn't functional in the doc recently in any case, so we will revisit this when we find a good example for a custom theme.

Reply:
I've just found other bugs to the new typedoc custom themes, you can see the new issue here: https://github.com/TypeStrong/typedoc/issues/1786

Reply:
@matteobruni thank you for the pointer, much appreciated!